src/rules/at-else-closing-brace-newline-after/__tests__/index.js

```diff
@@ -80,6 +89,11 @@ testRule(rule, {
 
       } @else if { }
       @else { }
+    }`,
+      fixed: `a {
+      @if ($x == 1) {
+
+      } @else if { }@else { }
```

Comment:
Not sure this fix makes sense 😕

Reply:
In my opinion `} @else if { } @else { }` would look better :)

Reply:
Makes sense, that's the same approach as in #221 👍